index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>News app</title>
+  </head>
+  <body class="wrapper">
+    <header class="header-wrapper">
+      <h1 class="logo">News Portal</h1>
+      <nav class="nav-wrapper">
+            
+      </nav> 
+    </header>
+    
+    <main class="content">
+        <div class="news"></div>
+    </main>
+
+    <footer class="footer">
+        <p class="copyright">Copyright <a class="copyright-link" target="_blank" href="https://newsapi.org">NewsAPI</a></p>
+    </footer>
+
+    <template id="navItemTemplate">
+      <div class="dropdown">
+        <button class="dropbtn"></button>
```

Comment:
Not sure if you need buttons for letters, this is rather list
Plus you have blue outline when you click the button

Reply:
done